Cargo.lock

```diff
@@ -102,9 +102,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.56"
+version = "1.0.57"
```

Comment:
Why the change to lockfile?

Reply:
So that's been happening to me sometimes and not sure why (especially when I'm cutting a release). I usually just unstage `Cargo.lock`, restore it, and then rebuild.